client/stylesheets/_responsive.scss

```diff
@@ -205,6 +290,27 @@
     // .profileCardLi {
     // 	width: 43% !important;
     // }
+    .searchResultsCardPrice {
+        margin: 6px 0 !important;
+    }
+
+    .searchResultsCardPrice span {
+        font-size: 0.9rem;
+    }
+
+    .searchResultsCardName p {
+        font-size: 0.8rem !important;
+    }
+
+    .searchResultsCardSize, 
+    .searchResultsCardCondition, .searchResultsCardLocation {
+        font-size: 0.9rem !important;
+        margin: 4px 0 !important;
+    }
+
+    .searchResultsCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardImg` should be written in lowercase with hyphens